torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2011,6 +2030,36 @@ RegisterOperators reg2({
           push(stack, std::sqrt(a));
           return 0;
         }),
+    DEFINE_GENERIC_PYTHON_OP(
+        aten::isnan,
+        isnan(a, b),
+        std::isnan(a / b),
+        float),
+    DEFINE_GENERIC_PYTHON_OP(
+        aten::asinh,
+        asinh(a, b),
+        std::asinh(a / b),
+        float),
+    DEFINE_GENERIC_PYTHON_OP(
+        aten::atanh,
+        atanh(a, b),
```

Comment:
Looks like you defined your macro to work on a single value, but here you are using it with two inputs `a` and `b`